enhancements/cloud-integration/azure/azure-workload-identity.md

```diff
@@ -0,0 +1,370 @@
+---
+title: azure-workload-identity
+authors:
+  - abutcher
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@2uasimojo"
+  - "@derekwaynecarr, for overall architecture."
+  - "@sdodson, for overall architecture."
+  - "@jharrington22, for service delivery considerations."
+  - "@RomanBednar, for azure file/disk operators."
+  - "@joelspeed, for MAPI / machine api operator."
+  - "@dmage, for image registry operator, please look at resource group being removed from credential secret and lookup from infrastructure object."
+  - "@Miciah, for ingress operator."
+  - "@patrickdillon, for installer."
```

Comment:
Tagging reviewers. PTAL at the proposal and let us know if you have any concerns with respect to providing workload identity details to operators / the installer. Thank you! :bow: 
@RomanBednar 
@dmage 
@Miciah 
@patrickdillon 
@r4f4 Adding you for the installer side as well since you've been spear heading and are familiar with the adal -> azidentity migration.

Reply:
@flavianmissi